src/components/Mapper/Mapper.js

```diff
@@ -17,6 +57,30 @@ function Mapper({ position, zoom }) {
         position="topright"
         sticky
       />
+      <FeatureGroup ref={localStorage.getItem('polygonCoords') ? (reactFGref) => featureGroupReady(reactFGref) : ''}>
+        <EditControl
+          position="topright"
+          onEdited={(e) => {
+            e.layers.eachLayer((a) => {
+              this.props.updatePlot({
+                feature: a.toGeoJSON(),
+              });
+            });
+          }}
+          onCreated={(e) => {
```

Comment:
This function might be refactored, it's understandable but, you could build it more clearly